Remove x-directory/normal from MimeType or else it replaces Nautilus as
default Places handler.

http://bugs.gentoo.org/show_bug.cgi?id=272198

diff -ur easytag-2.1.6.orig/easytag.desktop easytag-2.1.6/easytag.desktop
--- easytag-2.1.6.orig/easytag.desktop	2007-11-28 10:54:05.000000000 +0200
+++ easytag-2.1.6/easytag.desktop	2009-06-06 21:57:57.000000000 +0300
@@ -21,6 +21,4 @@
 Terminal=false
 Type=Application
 Categories=AudioVideo;AudioVideoEditing;GTK;
-X-Desktop-File-Install-Version=0.14
-MimeType=audio/x-mp3;application/ogg;x-directory/normal;
-
+MimeType=audio/x-mp3;application/ogg;
